Installed/Driver/Mouse/PS2/Makefile

```diff
@@ -1,61 +1,32 @@
-##############################################################################
-#
-# 	Copyright (c) GeoWorks 1992 -- All Rights Reserved
-#
-# PROJECT:	PC GEOS
-# MODULE:	PS2 -- Makefile
-# FILE: 	Makefile
-# AUTHOR: 	God, Wed Apr  2 20:26:53 PST 1997
-#
-# TARGETS:
-# 	Name			Description
-#	----			-----------
-#	ps2ec.geo		create an error-checking version
-#	ps2.geo		create a non error-checking version
-#	full			create both of the above, plus tags
-#	part			create only the error-checking version
-#	tags			create a tags file from all the sources
-#	depend			generate source file dependencies
-#
-# DESCRIPTION:
-#	This makefile was automatically generated from:
-#	    /staff/pcgeos/Tools/mkmf/Makefile.geode
-#
-#	DO NOT EDIT IT, AS YOU MAY LOSE YOUR CHANGES WHEN IT IS REGENERATED
-#
-#	$Id: Makefile.geode,v 1.4 92/01/03 20:17:45 johnm Exp $
-#
-###############################################################################
-
-GEODE           = ps2
-LIBNAME         = mouse

```

Comment:
This is missing in the regenerated Makefile. Because the Makefile should not cover any changes, please revert the file/restore its original.